editor/inserter/menu.js

```diff
@@ -245,12 +276,13 @@ class InserterMenu extends Component {
 
 	switchTab( tab ) {
 		this.setState( { tab: tab } );
+		this.setSearchFocus();
```

Comment:
There again, that was before search was at the top. I'll leave the call to someone who knows better!